en/05_Contributing/05_Coding_Conventions/01_PHP_Coding_Conventions.md

```diff
@@ -136,6 +136,133 @@ class MyClass extends Class
 }
 ```
 
+## Code style
+
+- Prioritise the human readability of code. Avoid "clever" code that aims to reduce the number of lines at the expense of readbability. This is particularly important when writing Javascript.
```

Comment:
```suggestion
- Prioritise the human readability of code. Avoid "clever" code that aims to reduce the number of lines at the expense of readability. This is particularly important when writing JavaScript code.
```

Reply:
I'm fine with using the word "clever" here. If people start arguing what is "clever" and what isn't, we can revisit.